resources/js/humhub.wiki.Form.js

```diff
@@ -74,9 +81,60 @@ humhub.module('wiki.Form', function(module, require, $) {
         }, 500);
     };
 
+    Form.prototype.openUrlLink = function(evt) {
+        var form = this.$;
+        form.attr('action', evt.$trigger.data('action-click-url'))
+            .submit();
+    }
+
     Form.submit = function () {
+        if (editPollingTimer) {
+            clearInterval(editPollingTimer);
+        }
         $('form[data-ui-widget="wiki.Form"]').submit();
     };
 
+    function pollTimerEditingStatus() {
+    
+        var url = document.querySelector('[data-url-editing-timer-update]').getAttribute('data-url-editing-timer-update');
+
+        client.get(url).then(function(response) {
+        }).catch(function(e) {
+            module.log.error(e, true);
+        });
+    }
+
+    function startEditPolling() {
+        if (editPollingTimer) {
+            clearInterval(editPollingTimer);
+        }
+        editPollingTimer = setInterval(pollTimerEditingStatus, editPollingInterval);
+    }
+
+    function checkValidUser() {
+        var url = document.querySelector('[data-url-editing-timer-update]').getAttribute('data-url-editing-timer-update');
+        var new_page = document.querySelector('[data-url-editing-timer-update]').getAttribute('data-new-page-entry');
+        if(!new_page) {
+            client.get(url).then(function(response) {
+                if(response.success&&response.unAuthorizedLogin) {
+                    var options = {
+                        'header': 'Confirm Edit',
+                        'body': response.user +' is already editing.<br> Editing it would cause conflict Do you really want to continue?',
```

Comment:
Instead of the username, the display username (first name + last name) should be shown.